### PR TITLE
Checkout: Make it clear before billing step that taxes may be added

### PR DESCRIPTION
**Pull request by sirbrillig**
## Proposed Changes

When you enter checkout, if the shopping cart does not yet have a tax location set, the taxes will not yet be calculated and will not be displayed nor included in the total price. For some customers, particularly those in countries outside the US where price-exclusive taxes are uncommon, this may be misleading.

In this PR we add "Tax: to be calculated" as a note where the taxes would normally be displayed when we do not yet know if we will add taxes or not.

Fixes https://github.com/Automattic/payments-shilling/issues/2575

Before             |  After
:-------------------------:|:-------------------------:
Screenshot 2024-05-28 at 6 05 13 PM | Screenshot 2024-05-28 at 6 10 28 PM
Screenshot 2024-05-28 at 6 05 17 PM | Screenshot 2024-05-28 at 6 27 31 PM
Screenshot 2024-05-28 at 6 15 29 PM | Screenshot 2024-05-28 at 6 12 51 PM

Because the mini-cart does not mention taxes normally, this PR also adds a notice to the mini-cart when the taxes have been added.

Screenshot 2024-05-28 at 6 18 09 PM

Note that if taxes are calculated and there is no tax applied, you will instead see a regular tax line.

Screenshot 2024-05-28 at 6 17 22 PM

## Testing Instructions

First make sure that you don't have taxes already saved on your shopping cart. A simple way is to use an account that has never made a purchase. If you are using an account that has purchased something before, this will require disabling the billing info autocomplete in checkout and any tax data already in the cart which you can do by sandboxing the API and applying the changes in D142087-code.

- Next, add a product to your cart and visit checkout.
- Verify that you see "Tax: to be calculated" displayed in the sidebar and the footer.
- **If you broke the endpoints as mentioned above, you will need to revert those changes now.**
- Change the billing info step to a location that does not charge taxes (eg: UAE).
- Verify that taxes _are not present_ and "Tax: to be calculated"  is NOT displayed.
- Change the billing info step to a location that charges taxes (eg: US, 10001)
- Verify that taxes _are present_ and "Tax: to be calculated"  is NOT displayed.

### CODE REVIEW

**Review by niranjan-uma-shankar — COMMENTED** [review]
Thank you for working on this.

Some notes:

1. This is probably already planned after the code is approved - I think we need a `String freeze` label to wait for the translations?
2. A `text-wrap: pretty` CSS prop could help prevent the orphan:
Screenshot 2024-03-14 at 12 20 13 PM

3. Wondering if we should "Estimated total" in the header in mobile view, for those users who don't expand the purchase items accordion. What do you think?
Screenshot 2024-03-14 at 12 21 36 PM

4. Probably in this PR or a different one, should we also add the "Estimated total" copy to the shopping cart in Calypso pages?
Screenshot 2024-03-14 at 12 29 25 PM

Even with tax calculated, I think this shopping cart view needs to mention that the total includes taxes. In the screenshot below, the price is listed as €357 but it does not mention the taxes.
Screenshot 2024-03-14 at 12 33 57 PM

**Comment by DavidRothstein** [comment]
I'm very skeptical of "Estimated total"... It's not an estimate, more like the minimum _possible_ total (most people will pay more).  I don't know what it should say, but maybe something like "Current total"?  Or just leave it as "Total"?

I'm also not sure "Taxes are calculated after billing information is provided" is accurate regarding how we actually handle taxes in all countries (for example, even after billing information is provided, in some cases we can show $0 tax and in other cases we can show no tax line at all, and I don't think those mean the exact same thing).  Maybe run this wording by the tax team to make sure it's OK in all cases?

**Comment by sirbrillig** [comment]
> I'm very skeptical of "Estimated total"... It's not an estimate, more like the minimum possible total (most people will pay more). I don't know what it should say, but maybe something like "Current total"? Or just leave it as "Total"?

My intent with this PR was to help our EU customers understand that they are likely to be paying more than is shown. I think that "Current total" or "Total" do not convey that message sufficiently, even when the "Taxes will be calculated..." text is also shown. That said, I'm very open to other options. I personally don't love how large the word "Estimated" is in the UI. Maybe "Total before taxes"? 

Or maybe we could add the subtext about "Taxes will be calculated..." under "Total" instead of in the taxes spot?

> (for example, even after billing information is provided, in some cases we can show $0 tax and in other cases we can show no tax line at all, and I don't think those mean the exact same thing)

This PR should already account for those cases. The text should be hidden once the billing information is provided or if no billing information is needed. 

Some examples of how this PR looks when taxes are 0 or are not collected:

Taxes in the UAE:

Screenshot 2024-03-14 at 11 57 38 AM

Taxes in New Hampshire, US:

Screenshot 2024-03-14 at 11 59 20 AM

> I'm also not sure "Taxes are calculated after billing information is provided" is accurate regarding how we actually handle taxes in all countries

What if we changed the wording to "Taxes may be applied once billing information is collected" to hint that taxes may not be needed?

**Comment by sirbrillig** [comment]
@DavidRothstein what do you think of this variation?

v1:

Screenshot 2024-03-14 at 12 13 48 PM

v2:

Screenshot 2024-03-14 at 12 14 59 PM

**Comment by sirbrillig** [comment]
> Wondering if we should "Estimated total" in the header in mobile view, for those users who don't expand the purchase items accordion. What do you think?

That's an interesting idea. 🤔 This is difficult because there's very limited space on that line. I'll have to check with @JanaMW27 to see what she thinks.

> Probably in this PR or a different one, should we also add the "Estimated total" copy to the shopping cart in Calypso pages?

Good idea, @niranjan-uma-shankar. Going along with the variation I suggested in my last comment, it might look like this:

Screenshot 2024-03-14 at 12 52 37 PM

> Even with tax calculated, I think this shopping cart view needs to mention that the total includes taxes. In the screenshot below, the price is listed as €357 but it does not mention the taxes.

That's true. What if we wrote something like "Includes applicable taxes"?

Screenshot 2024-03-14 at 1 00 20 PM

**Comment by jjchrisdiehl** [comment]
I think adding 'Total before tax' is the most common and succinct way of doing it. We could even show it in the sidebar in checkout until they add their billing information, then remove the 'before tax' wording after. Or even add some sort of language in its place like  **Total** _final price_.

**Comment by DavidRothstein** [comment]
The "Taxes may be added once billing information is provided" option underneath looks accurate to me; it feels a little clunky but I'm not sure I have a better wording suggestion.

It's _possible_ the "Total before tax" option is fine too; I'm just not sure it's an accurate description in all jurisdictions.  We do already say things like "excluding taxes" elsewhere, though (on the Plans page, etc).

I'd still suggest running the various options by the tax team to see what they recommend!

For comparison (and in case it helps jog any ideas) this is what we say when quoting prices in billing emails:

>This price does not include any applicable taxes which will vary based on your billing address.

**Comment by sirbrillig** [comment]
Updated screenshots and description to match the new design and I've pinged the Tax team for a review of the wording.

**Comment by Philanco** [comment]
> Updated screenshots and description to match the new design and I've pinged the Tax team for a review of the wording.

I think the wording should be more definitive, such as "Excludes applicable taxes." Could also be something like "Excludes applicable taxes calculated at checkout." 

The point is that the total amount shown does not include taxes, which most countries require to be explicitly stated.

**Comment by sirbrillig** [comment]
Hm... we could say `Excludes applicable taxes` but my hope was to suggest that we will (maybe) include taxes as soon as we have the information to do so, which is why I went with `Taxes may be added once billing information is provided`. But if you think that the former is more clear, then we can definitely go that route.

**Comment by pierorocca** [comment]
Thanks @sirbrillig for the opportunity to cross org lines to comment on this as it caught my attention. Per our Slack discussion, checkout works great and I'm not seeing a strong reason to make this change.

Rationale:

1. Checkout as-is looks completely fine and aligned with approaches used by top retailers and competing platforms including Shopify

2. Known signed-in shoppers WILL have a billing location on checkout page load.

3. Checkout uses a stepped / accordion design. Shoppers must progress through each section sequentially. A shopper can’t pay until the billing and payment information is provided, until which time the Pay button is in a disabled state. There's no path for a shopper to accidentally make a purchase with a non-taxed price total.

3. For guest shoppers, once the billing information is known, the appropriate tax label and amount becomes visible in the order summary AND in the summary above the Pay button. 

4. Anytime the order summary changes, there's a very visible skeleton loading state that draws attention to a change. See video.

5. Focus the user on getting through checkout to drive the conversion without distractions. A third reference to taxes on one page seems redundant.

6. Adding a message about taxes, if it even gets read, it's telling the shopper the price may be higher unnecessarily. No one likes the feeling of being told they're going to be taxed. 
 
**Clearly visible taxes in the order summary and summary above Pay button**

image

image

image

image

**Video of highly visible skeleton loading component when tax change occurs** 

https://github.com/Automattic/wp-calypso/assets/109928953/78fb74ce-3c61-48c3-a2cc-9efdf5fcbc37

**Comment by pierorocca** [comment]
The only thing I would suggest is that the skeleton component is enlarged so that it better approximates the actual size of the order summary and to minimize the visual movement.

**Comment by pierorocca** [comment]
Here's a better screen capture from the video showing actually a few things that are a problem:

1. As stated the skeleton size is too small relative to the actual size of the order summary creating a lot of up/down visual movement

2. New - there's no skeleton on the subtotal, tax, and total even though the values get updated. It looks like the skeleton is being applied to the wrong elements?

image

**Comment by DavidRothstein** [comment]
>A third reference to taxes on one page seems redundant.

@pierorocca I might be misunderstanding, but I thought everything being proposed in this pull request would only appear when there are **no** other references to taxes on the page yet.  Once the real taxes appear, the goal is for this to disappear.

**Comment by DavidRothstein** [comment]
>but my hope was to suggest that we will (maybe) include taxes as soon as we have the information to do so, which is why I went with `Taxes may be added once billing information is provided`

@sirbrillig Do you think we can communicate that part of it inside the billing information section itself?

It never entirely made sense to me that our description in that section says "Entering your billing information helps us prevent fraud" when that's not really the main reason (or at least not the only reason) that we need to collect it.  (I realize that the description is already somewhat different when a domain is in the cart, though.)

**Comment by sirbrillig** [comment]
Per the above conversation I'm going to move this back to draft status. 

The tax team's advice above suggests that we don't need to display any additional info in checkout about taxes not being calculated since, in most cases, it's clear that tax will be and is added to the prices (when appropriate).

There are some features in this PR that I do think are worth keeping, though, which will probably become their own PRs, particularly displaying tax inclusion/exclusion in the mini-cart.

**Comment by pierorocca** [comment]
> @pierorocca I might be misunderstanding, but I thought everything being proposed in this pull request would only appear when there are no other references to taxes on the page yet. Once the real taxes appear, the goal is for this to disappear

Good point @DavidRothstein. That's true for a point in time. Looking at the duration of the customer journey and applying pricing theory, specifically something called _prospect theory_ where people evaluating a transaction place more weight on "losses" than on equal sized "gains", it adds a third occurence of messaging a "loss". 

Think of a purchase where shipping isn't free, something most people have come to expect. That "loss" can outweigh a lot of other positives and cause abandonment. It's psychologically uncomfortable. What if during your next Amazon Prime checkout there were three messages in total telling you shipping wasn't free and was going to cost you $30? In a social situation if you told your partner that you loved their outfit despite it making them look chubby, would they be able to get past the chubby comment and take the compliment? (extreme example to illustrate this phenomenon)

Breaking this checkout journey into gains and losses:

Gains - The hosting service, the included features, the discount for the two year option
Losses - The price, and taxes

VAT taxes in EU countries average over 20% with the highest at 27%. That's no joke at up to 81 euros on a 300 euros purchase. Meeting the minimum requirement for disclosure, without shining a spotlight on this loss is my recommendation here.

**Comment by pierorocca** [comment]
@sirbrillig double checking this https://github.com/Automattic/wp-calypso/pull/88493#issuecomment-2000394102 didn't get lost in the thread. I think it's a loading component bug as the skeleton is not being applied to the values that do change but is to the static "included" items.

**Comment by niranjan-uma-shankar** [comment]
> and aligned with approaches used by top retailers and competing platforms including Shopify

@pierorocca I am not sure this is true - in fact I have seen the opposite. I checked a few competitors including Shopify - they all show the "estimated taxes" or some form of it. I haven't yet come across a competitor who either doesn't pre-calculate the tax or doesn't use the "estimated tax" language either directly or some form of it, and WP.com (and other products like Jetpack etc.) seems to be the exception here. In all the examples below, I neither have added my card info or billing contact info. 

**GoDaddy**
312503866-cddc6676-8c84-4700-af9e-fb0fbd0a890a

**Squarespace**

I'm currently in Spain, it pre-calculated the tax by auto-detecting my location and shows the "estimated total"

Screenshot 2024-03-19 at 5 19 38 PM

**Wix**

Auto-detected location and pre-calculated tax

Screenshot 2024-03-19 at 5 21 55 PM

**Webflow**
Screenshot 2024-03-19 at 5 23 36 PM

**Shopify**
Screenshot 2024-03-19 at 5 25 06 PM

**Siteground**

Pre-calculated tax by auto-detecting location

Screenshot 2024-03-19 at 5 26 30 PM


> Known signed-in shoppers WILL have a billing location on checkout page load.

I believe only those who have already saved a card with country info will have tax pre-calculated. None of the new user purchases will have this, up until the billing contact step is completed.

> There's no path for a shopper to accidentally make a purchase with a non-taxed price total.

That's true but the issue here is not about accidental payment which is not possible, as you pointed out. When a user loads the checkout, the total price is displayed prominently, but the tax is silently added after the contact info is added, which seems like could be easy to miss until the user sees the price jump on the CTA. It's not going to lead to an accidental purchase but probably could be annoying for some users? Maybe worth a test or hearing anecdotal evidence from EUmatticians who find this experience discordant with other sites they encounter.

> Adding a message about taxes, if it even gets read, it's telling the shopper the price may be higher unnecessarily. No one likes the feeling of being told they're going to be taxed.

We already mention quite prominently in the pricing grid in landing pages and signup flow that prices are exclusive taxes. The idea here is to normalise the experience in checkout. I am not sure if we can confidently conclude that making the message about tax harder to grasp could be a net positive for us and maybe that's something that could be an experiment, but probably not at the cost of trust in the brand. Having said that, I don't come from a country where showing tax prominently is standard practice (similar to the USA), and I am purely going by the feedback I have heard from many EUmatticians who seem to feel more strongly about this, an example is this post pdgrnI-27y-p2. 

> New - there's no skeleton on the subtotal, tax, and total even though the values get updated. It looks like the skeleton is being applied to the wrong elements?

Good catch and I agree with this 👍

**Comment by niranjan-uma-shankar** [comment]
> Meeting the minimum requirement for disclosure, without shining a spotlight on this loss is my recommendation here.

@pierorocca I am not entirely convinced by this. I agree with everything you said about prospect theory but it shouldn't be at the cost of trust in the brand. We definitely don't want the perceived losses (with added tax) to get prominence or spotlighted, but we also don't want users to perceive us as "shady" by hiding that information. Like I said earlier, most if not all our competitors mention taxes are separate, and we also wouldn't want to be among the few that don't. We could play around with the positioning of the text, where it should be shown, font size etc. to ensure that the copy doesn't get undue attention. 

As an example, there is a general distrust about money back guarantees in many parts of the world (check peRNfl-en-p2/#cancellation-and-refund about how difficult it is to get a refund on GoDaddy), but WP.com is among the few that really stands by this promise. This is a huge asset for us, but some small things could add up to users not trusting our other claims. 


> That's no joke at up to 81 euros on a 300 euros purchase. 

And that's probably why we need to be careful about not making this too obscure before billing contact is added (and at the same time also not make this a prominent thing on the page). There should be a way to strike a balance and we should also consider local customs and expectations (like maybe exclude USA and show it in the EU only). I think it's really hard to have a theoretical debate on the right approach and we can get a convincing answer if we can run a controlled experiment to see if showing taxes is affecting purchase conversions. I would recommend that we implement an experiment if there's concerns about showing any wording about tax could be a net negative for us.

**Comment by pierorocca** [comment]
@niranjan-uma-shankar yes agree that when the shipping or billing address is known or detectable, estimating or including the tax amount in the order summary is great. In those cases, the additional copy is unnecessary. 

Regarding the subscriptions terms language, that is 100% required and agree with that. 

I'm specifically addressing the PR use case of adding a note that "taxes may be added once the billing is provided" below the total or anywhere else inline in the form as it's incremental to the existing two tax elements - order summary and the second order summary above the pay button. Bear with me.

A couple of considerations: 

1. In Europe product prices, including shipping, tend to be inclusive of taxes. If they're not, there's an expectation that shoppers are informed.

2. In North America, product prices and shipping prices are never are inclusive of taxes. Shoppers expect taxes to be added and calculated in checkout and always after the shipping (or billing) address is entered. In Canada taxes vary by province. In the US, by city or county.

Go Daddy is a good example because the tax element has a placeholder message that's shown until the value is known. This model could work for European and North American norms. It leans towards the N.A. model but is neutral enough imo.

Shopify (Gymshark) and Ikea take a different approach where the tax elements appear only after the address is known, and without additional messaging on the screen. Shopify specifically tailors the UI pattern for regional norms mentioned above:

**Gymshark Europe - address not known**
image
 
**Gymshark Europe - address known** (note shipping cost includes taxes)
image

**Gymshark US - address not known**
image

**Gymshark US - address known**
image

Note: Not shown here but on Shopify's new one page checkout, tax calculation is not dependent on a postal code/full shipping address for countries that have a national VAT. They simply calculate the tax based on the country geolocation, avoiding the issue altogether. I'd like to reach out to the Woo Shipping and Tax team to see if this is something they'll consider supporting...probably needs Core support as well. It would improve all checkouts on the platform.

An interesting thing I'm seeing in Europe is that retailers hold the Total sale price constant across Euro countries. They adjust the taxes by shipping country to what they're legally required to collect and then adjust the base price to keep the total price constant. e.g. In Spain the 59.99 euro shirt has 11.24 in taxes. The same 59.99 euro shirt in Italy has 11.63 in taxes (value is incorrectly calculated).

**Ikea Spain - address unknown** 
https://github.com/Automattic/wp-calypso/assets/109928953/7b7c53d5-e811-4915-9861-956c54a9f734

Maybe it's me, I avoid adding new elements if possible and try to reuse existing elements or rely on affordances. "Excl. taxes" or "Excluding taxes" is an ok fallback.

**Comment by DavidRothstein** [comment]
>We already mention quite prominently in the pricing grid in landing pages and signup flow that prices are exclusive taxes. The idea here is to normalise the experience in checkout.

I think that's a key point to emphasize -- our current experience (at least for WordPress.com plans) is to show something like "$96 billed annually, excl. taxes" on the plans page, but then when the customer gets to checkout they just see $96 and no longer any mention of taxes at all.  The goal of this issue would be to make that experience more consistent.

**Comment by Philanco** [comment]
Could we do something like have Taxes with a note they are calculated when billing information is added? Something like here where shipping shows "calculated at next step"

Screenshot 2024-03-26 at 12 08 47 PM

**Comment by jjchrisdiehl** [comment]
> Could we do something like have Taxes with a note they are calculated when billing information is added? Something like here where shipping shows "calculated at next step"

On the technical side, yes we could check if any billing information has been added to the cart (or is already know via the shopping cart endpoint) and in the event there isn't any we could show a placeholder like "calculated once billing information is added". 

Then once the billing information step is completed we could update the placeholder with the applicable taxes. This feels more natural to me than "taxes may be calculated..." since this comes off sort of like we don't know? 

For instance, if it turns out they do not owe any tax then we could just display "tax: $0" which has the benefit of visually confirming that they will pay 0 taxes (rather than just not showing anything).

**Comment by JanaMW27** [comment]
This was included in the designs for the final version:

Screenshot 2024-03-27 at 09 27 26

Figma file: https://www.figma.com/file/sHxWnkrog5i873GIz7opcY/Checkout?type=design&node-id=907%3A59795&mode=design&t=8GEVjv2kTSSCei75-1

**Comment by DavidRothstein** [comment]
Keep in mind that the checkout steps only refer to it as "contact information" when there's a domain in the cart (otherwise it's "billing information").  So it may be confusing to have the sidebar refer to it as either of those directly, but especially confusing to refer to it as "contact information".

I'd suggest one of these:
- "Tax: Based on your billing address" (similar language as our billing emails)
- "Tax: To be calculated"

The second may be fine; keep in mind that the billing/contact information section will already be open when these users enter checkout, so as noted in https://github.com/Automattic/wp-calypso/pull/88493#issuecomment-2001365572 any details about using the provided address to calculate tax could just be shown there instead.  We don't necessarily need prominent details about how the tax is calculated in the sidebar also.

>For instance, if it turns out they do not owe any tax then we could just display "tax: $0" which has the benefit of visually confirming that they will pay 0 taxes (rather than just not showing anything).

We already show "Tax: $0" in that scenario; the situation where we show no tax line at all is a different one (see screenshots in https://github.com/Automattic/wp-calypso/pull/88493#issuecomment-1997762262) and my understanding is that they're intentionally distinguished (though not 100% sure)?  I agree it might be disconcerting to show "Tax: (some message about how taxes weren't calculated yet)" and then have that line disappear afterwards though.  I wonder if in these situations we could display "Tax: Not collected" after the address is entered; would that be accurate?

**Comment by niranjan-uma-shankar** [comment]
>  I wonder if in these situations we could display "Tax: Not collected" after the address is entered; would that be accurate?

I think from the user's perspective, they're still charged nothing ($0) so maybe Tax: $0 is okay? "Not collected" could give the impression that we might collect it later? Or maybe "Tax: Not applicable" could work too.

**Comment by DavidRothstein** [comment]
@Philanco maybe you can help with the above part?  Currently for locations like New Hampshire, Ohio, etc., we show "Tax: $0" in checkout, but for locations like the United Arab Emirates we show no tax line at all.  If we want to add a tax line for the latter (United Arab Emirates) case, which of these would be OK to use?
1. "Tax: $0"
2. "Tax: Not collected"
3. "Tax: Not applicable"
4. Something else?

Thanks.

**Comment by niranjan-uma-shankar** [comment]
Outside of the [website builder and ecommerce examples](https://github.com/Automattic/wp-calypso/pull/88493#issuecomment-2007666553), I saw that X.com checkout contains the tax info before I input my billing address, and it auto-detected my location (that's why tax is shown as $0):

Screenshot 2024-04-22 at 11 39 19 AM

**Comment by DavidRothstein** [comment]
We need to move this forward.  I suggest we go with the simplest option:

**"Tax: To be calculated"**

With this option, we can consider also adding information to the "Enter your billing information"/"Enter your contact information" checkout step (that explains there that the provided address will be used for tax calculation) but I don't think it's a strict requirement to do that either.

If we implement the above as-is, then certain customers (e.g. in the United Arab Emirates) would see "Tax: To be calculated" before entering their address, but the tax line would then randomly disappear entirely after they enter their address -- as opposed to most users who will see "Tax: To be calculated" replaced with "Tax: $X" (where X can be 0) after they enter their address.  If we can get some clarification on a better way to handle the United Arab Emirates case (see https://github.com/Automattic/wp-calypso/pull/88493#issuecomment-2033169183) we can do that here, but if not then I think we can just live with the introduction of that odd disappearing behavior for that particular small subset of users and file a followup issue about it instead.

**Comment by sirbrillig** [comment]
> "Tax: To be calculated"

I've rebased this PR and made the change you suggested. I've updated the screenshots in the PR description. Let me know what you think.

**Review by michaeldcain — APPROVED** [review]
This LGTM and works as described. If we need to iterate on the design, we can do it in a follow-up.

**Comment by sirbrillig** [comment]
> Thanks! Do we need to get this translated before deploying it?

That would be ideal, yes. I've added a String Freeze and will leave it to someone else on the team to merge when translations are complete (as I'll be AFK).

**Comment by DavidRothstein** [comment]
For some reason this PR was stuck for weeks waiting for translations.  I just rebased it now to see if that helps.

File: client/my-sites/checkout/src/components/wp-checkout-order-summary.tsx
Comment (michaeldcain) [inline client/my-sites/checkout/src/components/wp-checkout-order-summary.tsx]: This context was removed in a previous diff, but I think it helps given the limited space in Checkout.

**Comment by michaeldcain** [comment]
> For some reason this PR was stuck for weeks waiting for translations. I just rebased it now to see if that helps.

Still seems to be the case, and the PR needed another rebase after the removal of Checkout test code in #90780

**Comment by jjchrisdiehl** [comment]
Rebasing again as this PR had some sort of duplicate `.blaze-pro` style on line 19 of https://github.com/Automattic/wp-calypso/blob/trunk/client/layout/masterbar/blaze-pro.scss which doesn't exist in trunk.

**Comment by dlind1** [comment]
> > For some reason this PR was stuck for weeks waiting for translations. I just rebased it now to see if that helps.
> 
> Still seems to be the case, and the PR needed another rebase after the removal of Checkout test code in #90780, so I went ahead and rebased. I'll follow-up with @Automattic/i18n if the translations are still stuck.

We've had an issue with our a8ci18n bot token a few weeks ago and the translations in Calypso PRs weren't working properly. This has been fixed since, but the current PR might still be affected by that, so I've created the [deliverable](https://translate.wordpress.com/deliverables/overview/15334281/) and it'd be translated in a few days.

**Comment by DavidRothstein** [comment]
The translations are finally in!

I pushed up one small change to give one of the new components a more accurate name, and will plan to do some final testing of this later (and hopefully deploy it).

File: client/my-sites/checkout/src/components/payment-method-step.tsx
Comment (DavidRothstein) [inline client/my-sites/checkout/src/components/payment-method-step.tsx]: I'm going to add back a little bit of bottom padding here.  Changing it to 0 made the new text bunch together too much with the "Total" line above it.

File: client/my-sites/checkout/src/components/wp-checkout-order-summary.tsx
Comment (DavidRothstein) [reply client/my-sites/checkout/src/components/wp-checkout-order-summary.tsx]: I agree but I think if we do that we need to do it consistently, not just in this part of the code.  With this change in place and using German, I see the word "Total" translated two different ways ("Gesamt" and "Gesamtbetrag") in different parts of the same checkout screen.

I'm going to revert this and file a followup issue to implement it everywhere.

Comment (DavidRothstein) [reply client/my-sites/checkout/src/components/wp-checkout-order-summary.tsx]: Followup issue here: https://github.com/Automattic/wp-calypso/issues/92565

**Comment by DavidRothstein** [comment]
> If we can get some clarification on a better way to handle the United Arab Emirates case (see [#88493 (comment)](https://github.com/Automattic/wp-calypso/pull/88493#issuecomment-2033169183)) we can do that here, but if not then I think we can just live with the introduction of that odd disappearing behavior for that particular small subset of users and file a followup issue about it instead.

Tracking that in https://github.com/Automattic/payments-shilling/issues/2972 now.

>I kind of pictured the "To be calculated" to be aligned right, for what it's worth 🤷 But it's not a blocker.

Tracking that and a related potential styling followup in https://github.com/Automattic/wp-calypso/issues/92566 now.

Meanwhile, I'll plan to get this one deployed tomorrow.

**Pull request merged**
